server/app/views/components/ToastMessage.java

```diff
@@ -44,7 +44,8 @@ public static ToastMessage alert(String message) {
     return new ToastMessage(message, ToastType.ALERT);
   }
 
-  public static ToastMessage error(String message) {
+  public static ToastMessage error(String message, boolean isLocalized) {
+    if (!isLocalized) message = "Error: " + message;
```

Comment:
I prefer not to force clients to use TOAST_ERROR_MSG_OUTLINE themselves, if every error toast will use it. How about this implementation which does it on behalf of the clients, so files like `ApplicantProgramReviewController` don't have to?

```java
  public static ToastMessage error(String message, Messages messages) {
    String errMessageWithPrefix = messages.at(
      MessageKey.TOAST_ERROR_MSG_OUTLINE.getKeyName(), message);
    return new ToastMessage(errMessageWithPrefix, ToastType.ERROR);
  }
```

Reply:
This seems like a very good idea, although I'll have to modify it to accommodate all the admin error toasts which aren't using `play.i18n.Messages.`  I guess I could check if `messages` is null rather than having a third `isLocalized` param.

Reply:
For the admin error toasts, `ToastMessage.errorNonLocalized()` would not have a `Messages` param. So it would be a minimal change, for those, right?

Reply:
Applicant error toasts call `ToastMessage.error(String message, Messages messages)`

Admin error toasts call `ToastMessage.errorNonLocalized(String message)`

So no need for a isLocalized param. Make sense? Am I missing anything?

Reply:
Yep, makes a lot of sense.  I think that will work well.